installer/resources/pacbot_app/files/DB.sql

```diff
@@ -2746,6 +2746,7 @@ INSERT IGNORE INTO `cf_Target` (`targetName`,`displayName`, `targetDesc`, `categ
 INSERT IGNORE INTO `cf_Target` (`targetName`,`displayName`, `targetDesc`, `category`, `dataSourceName`, `targetConfig`, `status`, `userId`, `endpoint`, `createdDate`, `modifiedDate`, `domain`) VALUES('mysqlflexible','My SQL Flexible Server','Azure mysqlflexible','','azure','{\"key\":\"id\",\"id\":\"id\",\"name\":\"name\"}','enabled','admin@pacbot.org',concat(@eshost,':',@esport,'/azure_mysqlflexible/mysqlflexible'),'2019-09-05','2022-09-05','Infra & Platforms');
 INSERT IGNORE INTO `cf_Target` (`targetName`,`displayName`, `targetDesc`, `category`, `dataSourceName`, `targetConfig`, `status`, `userId`, `endpoint`, `createdDate`, `modifiedDate`, `domain`) VALUES('diagnosticsetting','Diagnostic Setting','Azure diagnosticsetting','','azure','{\"key\":\"id\",\"id\":\"id\"}','enabled','admin@pacbot.org',concat(@eshost,':',@esport,'/azure_diagnosticsetting/diagnosticsetting'),'2019-09-26','2022-09-26','Infra & Platforms');
 INSERT IGNORE INTO `cf_Target` (`targetName`,`displayName`, `targetDesc`, `category`, `dataSourceName`, `targetConfig`, `status`, `userId`, `endpoint`, `createdDate`, `modifiedDate`, `domain`) VALUES('vaultsrbac','Vaults with role base access control','Azure vaultsrbac','','azure','{\"key\":\"id\",\"id\":\"id\"}','enabled','admin@pacbot.org',concat(@eshost,':',@esport,'/azure_vaultsrbac/vaultsrbac'),'2019-11-10','2022-11-10','Infra & Platforms');
+INSERT IGNORE INTO `cf_Target` (`targetName`,`displayName`, `targetDesc`, `category`, `dataSourceName`, `targetConfig`, `status`, `userId`, `endpoint`, `createdDate`, `modifiedDate`, `domain`) VALUES('environment','Dast Assets','Running Instance','','checkmarx','{\"key\":\"id\",\"id\":\"id\"}','enabled','admin@pacbot.org',concat(@eshost,':',@esport,'/checkmarx_environment/environment'),'2023-08-07','2023-08-07','Infra & Platforms');
```

Comment:
I'd suggest giving current data for `createdDate`, `modifiedDate`

Reply:
@ershad-paladin Good catch. Modified wih curdate() function. Please check with latest push/commit